### PR TITLE
FIX: Parsing client IPv6 address when a proxy is involved

### LINKED ISSUE #456: BUG: IPv6 REMOTE_ADDR
This PR breaks compatibility with IPv6 remote addresses:
https://github.com/jazzband/django-auditlog/pull/417/files

I'll open a PR to fix it

### PR DESCRIPTION
**Pull request by aqeelat**
Fixes #456

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changes
 
+## Master
```
Comment (hramezani) [inline CHANGELOG.md:3]: Pleaes remove this line

**Comment by hramezani** [comment]
Thanks @aqeelat for this patch :+1: 
I left a small comment. Please address it. Then we are good to go

File: CHANGELOG.md (hunk 1 above)
Comment (alieh-rymasheuski) [reply CHANGELOG.md:3]: Can we reword this to "next release"? I think it's valuable to mark the topmost block of changes in the changelog as "unreleased yet, but will be soon".

Comment (hramezani) [reply CHANGELOG.md:3]: Yes, I would replace it with `unreleased` or `next release`

Comment (aqeelat) [reply CHANGELOG.md]: I updated it with Next Release

**Review by hramezani — APPROVED** [review]

**Comment by hramezani** [comment]
Thanks @aqeelat for the update.

> Do you have an ETA on when it might be released?

Probably end of this week I will release a patch release.
